index.js

```diff
@@ -1,123 +0,0 @@
-const readline = require("readline");
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  let name = await ask("What is your name?\n");
-  console.log("Hello " + name + " Lets Play a guessing game.");
-  let pickMode = await ask(
-    "Please chose your gaming mode : Player or Computer.\n"
-  );
-  if (pickMode == "player") {
-    startPlayer(); // Start Player Mode
-  } else if (pickMode == "computer") {
-    startComputer(); //Start Computer Mode
-  }
-}
```

Comment:
This is looking for specific inputs from the user and bugs out if you put in anything else and the computer game doesnt start up correctly